pkgs/development/compilers/gcc/common/platform-flags.nix

```diff
@@ -3,10 +3,13 @@
 let
   gcc = targetPlatform.gcc or {};
   p =  gcc
+  isAarch64Darwin = targetPlatform.isDarwin && targetPlatform.isAarch64;
```

Comment:
You probably need to move it out of the middle of `p` expression (1 line above or below) as eval fails as:

```
       error: syntax error, unexpected '=', expecting ';'

       at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-4/pkgs/development/compilers/gcc/common/platform-flags.nix:6:19:

            5|   p =  gcc
            6|   isAarch64Darwin = targetPlatform.isDarwin && targetPlatform.isAarch64;
             |                   ^
```

Reply:
Of course, thank you.